5calls/app/src/main/java/org/a5calls/android/a5calls/util/ScriptReplacements.java

```diff
@@ -15,6 +15,7 @@ public class ScriptReplacements {
     private static final Pattern SENATE_INTRO_PATTERN = Pattern.compile("\\*{2}WHEN CALLING SENATE:\\*{2}\\n");
     private static final Pattern CONTACT_NAME_PATTERN = Pattern.compile("\\[REP/SEN NAME]|\\[SENATOR/REP NAME]|\\[SENATOR NAME]|\\[REPRESENTATIVE NAME]");
     private static final Pattern LOCATION_PATTERN = Pattern.compile("\\[CITY,\\s?ZIP]|\\[CITY,\\s?STATE]");
+    private static final Pattern NAME_PATTERN = Pattern.compile("\\[NAME]");
```

Comment:
consider adding whitespace and let it be case insensitive. See PR which I will send for your review, https://github.com/5calls/android/pull/145